RandoWPF/Randomizer.cs

```diff
@@ -32,3 +34,26 @@ public virtual void Save()
         return new Dictionary<string, HTMLPage>();
     }
 }
+
```

Comment:
We can probably make the Plandomizer an interface and then have each plandomizer inherit from the randomizer and IPlandomizer.
That way we can reuse code in both that does not use the plando.
Ex: `class BattlePlandomizer : BattleRandomizer, IPlandomizer`